<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>684</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>687</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(209, 205, 255)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>410</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Sans Typewriter</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color:rgb(255, 255, 255);
background-color:rgb(181, 176, 255);
border-radius:30;
}

QPushButton:pressed {
background-color:rgb(163, 78, 194);
}</string>
    </property>
    <property name="text">
     <string>Начать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>410</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Sans Typewriter</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color:rgb(255, 255, 255);
background-color:rgb(181, 176, 255);
border-radius:50;
}

QPushButton:pressed {
background-color:rgb(163, 78, 194);
}</string>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>651</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Sans Typewriter</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white</string>
    </property>
    <property name="text">
     <string>Ответ вводите с большой буквы,</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>541</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Sans Typewriter</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white</string>
    </property>
    <property name="text">
     <string>Например:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>541</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Sans Typewriter</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white</string>
    </property>
    <property name="text">
     <string>Бромоводородная кислота - HBr</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>681</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Sans Typewriter</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white</string>
    </property>
    <property name="text">
     <string>без лишних пробелов и символов</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
